README.md

```diff
@@ -1 +1,71 @@
 # NYT-webpage-clone
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> This is my newyork clone website.
+
+![screenshot](index.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+- On the project GitHub page, navigate to the main page of the repository [this page](https://kunmi02.github.io/NYT-webpage-clone/).
```

Comment:
- [ ] [OPTIONAL] Your live link should be the star of the README file! 
Instead of hidding it midway put it at the very top, before the screenshot, and maybe also in the title, and make it very obvious that this is the live link. 
Strive for clarity when building README files and any other type of documentation.